versions/3.0.md

```diff
@@ -2974,10 +2974,13 @@ Supported schemes are basic authentication, an API key (either as a header or as
 ##### Fixed Fields
 Field Name | Type | Validity | Description
 ---|:---:|---|---
-<a name="securitySchemeType"></a>type | `string` | Any | **Required.** The type of the security scheme. Valid values are `"basic"`, `"apiKey"` or `"oauth2"`.
+<a name="securitySchemeType"></a>type | `string` | Any | **Required.** The type of the security scheme. Valid values are `"apiKey"`, `"http"`, `"oauth2"`, `"openIdConnect"`.
```

Comment:
The license to use/reference OpenID related works seems pretty unrestricted http://openid.net/specs/openid-connect-discovery-1_0.html#Notices 
To be more generic we could have a `IdentityProviderURL`.  OpenID provides both a WebFinger protocol and a .well-known URL for getting at the OpenID configuration.  If other Identity Providers come along they would probably have their own "discovery" mechanism.